app/views/contexts/summary/group.py

```diff
@@ -181,14 +200,19 @@ def _build_blocks_and_links(
                         location=location,
                         language=language,
                         return_to=return_to,
+                        return_to_block_id=return_to_block_id,
                     )
 
-                    list_summary_element = list_collector_block.list_summary_element(
+                    # Type ignore: base block check not required here as both child blocks have list_summary_element
+                    list_summary_element = list_collector_block.list_summary_element(  # type: ignore
```

Comment:
This is valid, you need to fix the typing for the above lines when initialisation the class to fix this.

Reply:
Fixed the type, ignore removed.